src/autoschedulers/anderson2021/State.h

```diff
@@ -122,15 +130,28 @@ struct State {
         void add_outer_thread_loops(LoopNest *loop_nest) const;
     };
 
-    IntrusivePtr<const LoopNest> get_root_for_features(const Anderson2021Params &params, const Target &target) const;
+    IntrusivePtr<const LoopNest> get_root_for_features(const Anderson2021Params &params,
+                                                       const Target &target) const;
 
-    void set_gpu_store_site(const map<const LoopNest *, pair<const LoopNest *, int>> &parent, const LoopNest *loop, LoopNest::Sites &site) const;
+    void set_gpu_store_site(const map<const LoopNest *,
+                                      pair<const LoopNest *, int>> &parent,
```

Comment:
same here

Reply:
Oh, good catch, I misread the comma in manual formatiing, willfix